src/users/store/actions/user.actions.ts

```diff
@@ -122,6 +125,21 @@ export class ReinvitePendingUser {
   constructor(public payload: any) {}
 }
 
+export class LoadAccessTypes {
+  public readonly type = LOAD_ACCESS_TYPES;
+  constructor(public payload?: any) {}
```

Comment:
We should strongly type here to know what to expect. I spoke with other team and strongly typing the parameters is the direction they're going too.